indexing-service/pom.xml

```diff
@@ -195,6 +195,22 @@
             <artifactId>commons-collections4</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.cronutils</groupId>
+            <artifactId>cron-utils</artifactId>
+            <version>9.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.druid</groupId>
+            <artifactId>druid-sql</artifactId>
```

Comment:
This was my initial plan, but that would entail moving Calcite dependencies along with several classes from the `sql` module because of how things are wired in now. Neither the `processing` module nor the `server` module currently depends on Calcite. 

That said, this new feature relies on the Broker to run explain plan queries, so perhaps adding this dependency for the client and serde classes in the `indexing-service` module is reasonable?

Reply:
No, we should definitely not add a Calcite dependency in these modules.
You might need to create some new interfaces and data model classes to make sure that the clients are able to talk to the Broker without worrying about the deps required by the actual implementation.

As for the Explain, I think we should remove that for now at least, and always force the user to pass the datasource name.